src/debugger/sourceMap.ts

```diff
@@ -15,7 +15,7 @@ interface ISourceMap {
 }
 
 export class SourceMapUtil {
-    private static SourceMapURLRegex: RegExp = /\/\/(#|@) sourceMappingURL=(.+?)\s*$/m;
+    private static SourceMapURLRegex: RegExp = /\/\/(#|@) sourceMappingURL=(.+index\.ios\.map.*?)\s*$/m;
```

Comment:
Ok, found it. This is an issues related to `react-intl`

https://github.com/yahoo/react-intl/issues/71